docs/ForecastQueryFormat.md

```diff
@@ -92,35 +92,31 @@ Some features (currently only automatic prediction type conversion) require addi
 ```
 
 
-### Automatic prediction type conversion
+### Prediction type conversion
```

Comment:
Could we add something in this section about how the conversion is requested? It's not clear to me what tool someone would use to activate the conversion of point types. Maybe this should be a separate issue.